message_command.py

```diff
@@ -57,7 +59,7 @@ def __init__(self, message: str, channel: discord.TextChannel, member: discord.M
         self.member_id   = member.id
         self.member_name = member.display_name
 
-        with codecs.open("messages.json", 'r', 'utf-8') as json_file:
+        with codecs.open(os.path.dirname(__file__) + "/messages.json", 'r', 'utf-8') as json_file:
```

Comment:
```suggestion
        with codecs.open(os.getcwd() + "/messages.json", 'r', 'utf-8') as json_file:
```
今気づいたんですけど同じコード2つあるんだなこれ
初めて知った